Livewire v3 + README updates

Edgarborras94 opened the pull request:
Just noticed some changed are from my formatter lmao, sorry about that @joshhanley (:

Changed also the readme to include an example of the add new event we talked about several months ago (better late than never ;) )

I'm glad contributing to open source as I'm getting confident with the Laravel ecosystem!!

joshhanley commented:
@Edgarborras94 Thanks! I've triggered the tests to run. Not sure if they will though as I'll probably need to update then to suit v3. 

Re styling, generally it's better to remove style changes as it's harder for maintainers to review the code you actually changed (vs code format changes). So if it's not too hard, can you reset the styling?

Also thanks for updating the readme! Will have a good look in the next couple of days 🙂

Yeah it's always nice to be able to contribute 😁

Edgarborras94 commented:
I've removed the reformatted code I think. Also noticed I removed the package "joshhanley/livewire-dusk-testbench": "^0.1.0" because it's still not v3 compatible (I think)

I will keep an eye on this topic so tell me anything you need

Cheers!

joshhanley commented:
@Edgarborras94 thanks for the PR! Sorry it took me a while to get to it.

Removing the `joshhanley/livewire-dusk-testbench` meant I couldn't confirm everything was working correctly, as no tests would run, hence why I delayed until I had time to dig in.

I've got tests running now without it, using a fork of Livewire which exposes some of it's dusk testing internals, which I hope to get merged into Livewire core so package maintainers can use it. If that doesn't work, then I will update `joshhanley/livewire-dusk-testbench` with the V3 code.

Thanks again for your help! 😁
